index.php

```diff
@@ -5,19 +5,37 @@
 <main id="main-content">
   <section id="posts">
     <div class="container">
-      <div class="grid-row">
 
 <?php
 if( have_posts() ) {
   while( have_posts() ) {
     the_post();
+
+    if (has_category()) {
+      $category = get_the_category();
+      $cat_name = $category[0]->cat_name;
+      $cat_id = get_cat_ID( $name );
+      $cat_link = get_category_link( $cat_id );
+    }
 ?>
 
-        <article <?php post_class('grid-item item-s-12'); ?> id="post-<?php the_ID(); ?>">
+        <article <?php post_class('grid-row'); ?> id="post-<?php the_ID(); ?>">
+
+          <div class="grid-item item-s-12 item-m-4">
+            <h1><a class="link-underline" href="<?php the_permalink() ?>"><?php the_title(); ?></a><h1>
+          </div>
+
+          <div class="grid-item item-s-12 item-m-4">
+            <time datetime="<?php the_time('Y-m-d'); ?>"><?php echo get_the_date(); ?></time><br>
```

Comment:
Also is there a difference in WP between the time and the date of a post? Sometimes I get some errors with one of these not returning in the loop when the same value as an adjacent post. Not sure exactly about these specific functions